articles/dms/howto-mysql-to-azure-mysql-powershell.md

```diff
@@ -0,0 +1,657 @@
+
+> [!IMPORTANT]
+> The “MySQL to Azure Database for MySQL” online migration scenario is being replaced with a parallelized, highly performant offline migration scenario from June 1, 2021. For online migrations, you can use this new offering together with [data-in replication](https://docs.microsoft.com/azure/mysql/concepts-data-in-replication). Alternatively, use open-source tools such as [MyDumper/MyLoader](https://centminmod.com/mydumper.html) with data-in replication for online migrations. 
+
+The article helps to automate the scenario where source and target database names can be different and as part of migration either all or few of the tables in the target database need to be migrated which have the same name and table structure. Although the articles assumes the source to be a MySQL database instance and target to be Azure Database for MySQL, it can be used to migrate from one Azure Database for MySQL to another just by changing the source server name and credentials. Also, migration from lower version MySQL servers (v5.6 and above) to higher versions is also supported.
+
+> [!INCLUDE [preview features callout](../../includes/dms-boilerplate-preview.md)]
+
+In this article, you learn how to:
+> [!div class="checklist"]
+>
+> * Migrate database schema.
+> * Create a resource group.
+> * Create an instance of the Azure Database Migration Service.
+> * Create a migration project in an Azure Database Migration Service instance.
+> * Configure the migration project to use the Offline migration capability for MySQL.
+> * Run the migration.
+
+## Prerequisites
+
+To complete these steps, you need:
+
+* Have an Azure account with an active subscription. [Create an account for free] (https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+* Have an on-premises MySQL database with version 5.7 or 8. If not, then download and install [MySQL community edition](https://dev.mysql.com/downloads/mysql/) 5.7 or 8.
+* [Create an instance in Azure Database for MySQL](../mysql/quickstart-create-mysql-server-database-using-azure-portal.md). Refer to the article [Use MySQL Workbench to connect and query data](../mysql/connect-workbench.md) for details about how to connect and create a database using the Workbench application. The on-premises MySQL version must match with Azure Database for MySQL version. For example, MySQL 5.7 can only migrate to Azure Database for MySQL 5.7 and not upgraded to 8. 
+* Create a Microsoft Azure Virtual Network for Azure Database Migration Service by using Azure Resource Manager deployment model, which provides site-to-site connectivity to your on-premises source servers by using either [ExpressRoute](../expressroute/expressroute-introduction.md) or [VPN](../vpn-gateway/vpn-gateway-about-vpngateways.md). For more information about creating a virtual network, see the [Virtual Network Documentation](../virtual-network/index.yml), and especially the quickstart articles with step-by-step details.
+
+    > [!NOTE]
```

Comment:
There is an updated text for this in our online scenario, just double check which one is correct and use that at both the places.

Reply:
Fixed the online document to replace portal with workbench which is correct